draft-ietf-ppm-dap.md

```diff
@@ -481,7 +481,7 @@ uint8 HpkeConfigId;
 struct {
   HpkeConfigId config_id;    // config ID
   opaque enc<1..2^16-1>;     // encapsulated HPKE key
-  opaque payload<1..2^16-1>; // ciphertext
+  opaque payload<1..2^32-1>; // ciphertext
```

Comment:
I think a 4-byte length prefix is overkill, given that we're never going to have a 4GB `payload` here. Can we change this to a 3-byte prefix so that we don't have a dead byte on the wire?
```suggestion
  opaque payload<1..2^24-1>; // ciphertext
```

Reply:
3 bytes can only support up to 16MB, it's not impossible to need more than that in some use cases, especially if we go beyond histogram collecting. Having said that, I think that will be rare cases.

Reply:
We're talking abut a single byte here, so I would just do 4 bytes. That would also keep implementations simpler (`uint32_t` is everywhere, but `uint24_t` is not really a thing).